### PR TITLE
Unify de_DE

### PR DESCRIPTION
**Pull request by SteWers**
## Description:
Make de_DE more unify

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [ ] The code change is tested and works with Tasmota core ESP32 V.2.0.14
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

**Review by Noschvie — APPROVED** [review]

File: tasmota/language/de_DE.h [hunk 1]
```diff
@@ -348,7 +348,7 @@
 #define D_MQTT_TLS_ENABLE "MQTT TLS"
 #define D_HTTP_API "HTTP API"
 #define D_HTTP_API_ENABLE "HTTP API aktivieren"
-#define D_FRIENDLY_NAME "Name [Friendly Name]"
+#define D_FRIENDLY_NAME "Friendly Name"
```
Comment (eku) [inline tasmota/language/de_DE.h:351]: Why not translate this one?

Comment (SteWers) [reply tasmota/language/de_DE.h:351]: Not everything should be translated as the are no better words in german. The previous expression breaks the two column design and doesn't say more than the new one.

**Comment by Jason2866** [comment]
Whats better with Einstellungen as Konfiguration? Later on Konfiguration is still used.
I see no unify.  Anyways idk really whether the one or the other word is used...

**Comment by SteWers** [comment]
That's philosophical. I don't know, which word is better, but for me Einstellungen feels better. At the underlying settings pages Einstellungen is also used. So it is unify. Only on save, restore and reset Konfiguration is used. I think, that it is the better word at that place.
But you're right. Out of the tasmota build, there are a few points not unify. I'll think about them.

**Pull request merged**